statsmodels/stats/pscore_match.py

```diff
@@ -0,0 +1,242 @@
+'''Propensity Score Matchig
+
+
+Author: 
+License: BSD-3
+
+'''
+
+import statsmodels.api as sm
+import numpy as np
+from statsmodels.stats.weightstats import CompareMeans, DescrStatsW
+import pandas as pd
```

Comment:
Most of our procedures accept bare numpy arrays as well as pandas structures.  Where possible, the low level details are usually numpy only.  If a user passes a Pandas structure we convert it using numpy.asarray and attach the original pandas structure as a private attribute.  I don't see yet if you are using any Pandas functionality that would not be easy to do in numpy.